api/database/migrations/2022_11_25_232602_would_accept_temporary_to_position_duration.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+use Database\Helpers\ApiEnums;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+class WouldAcceptTemporaryToPositionDuration extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->jsonb('position_duration')->nullable()->default(null);
+        });
+        Schema::table('applicant_filters', function (Blueprint $table) {
+            $table->jsonb('position_duration')->nullable()->default(null);
+        });
+
+        DB::statement(
```

Comment:
Should we drop the `would_accept_temporary` column after migrating the data? Or are we leaving it in for backwards compatibility?

Reply:
acceptance criteria was to deprecate without totally breaking it, so yeah I think it is staying for a bit